app/src/styling/pages/_landing.scss

```diff
@@ -0,0 +1,88 @@
+$body-bg: #FFDA7C;
+$purple: #6A4C93;
+
+*{
+    margin:0;
+    padding:0;
+    box-sizing: border-box;
+}
+
+body{
+    background-color:$body-bg;
+    display:flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    text-align:center
+}
+
+.landing-page{
```

Comment:
and when it comes to the page styling, this can be done easily with some css flex, properties like justify-content: space-evenly; can absolutely help us achieve the wanted result and above all else it will help us to make the page responsive even easier.